aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -1127,6 +1127,41 @@ Blazor Server:
 
 **In the preceding example, the `{BLAZOR SCRIPT}` placeholder is the Blazor script path and file name.** For the location of the script and the path to use, see <xref:blazor/project-structure#location-of-the-blazor-script>.
 
+:::moniker range=">= aspnetcore-9.0"
+
+When the user navigates back to an app with a disconnected circuit, reconnection is attempted immediately rather than waiting for the duration of the next reconnect interval to resume  the connection as quickly as possible for the user.
```

Comment:
```suggestion
When the user navigates back to an app with a disconnected circuit, reconnection is attempted immediately rather than waiting for the duration of the next reconnect interval to resume the connection as quickly as possible for the user.
```

Reply:
Looks like you already caught this one!

Reply:
I think that one I already fixed.